tomls/omnibus-arbitrum-mainnet/oracles/pyth-btc.toml

```diff
@@ -0,0 +1,52 @@
+# This registers a staleness node that looks to Pyth for a fresh enough price, or otherwise reverts with an ERC-7412 compatible error
+# Latest Pyth Price
+[invoke.registerPythBtcOracleNode]
+target = ["system.oracle_manager.Proxy"]
+func = "registerNode"
+args = [
+    5, # 5 = pyth aggregator type
+    "<%= defaultAbiCoder.encode(['address', 'bytes32', 'bool'], [settings.pyth_price_verification_address, settings.pyth_feed_id_btc, false]) %>",
+    [
+    ],
+]
+extra.btc_pyth_oracle_id.event = "NodeRegistered"
+extra.btc_pyth_oracle_id.arg = 0
+depends = [
```

Comment:
I believe after cannon fixed something we no longer need these depends. Could you check if it works without please